MekHQ/unittests/mekhq/campaign/stratcon/StratconRulesManagerTest.java

```diff
@@ -0,0 +1,12 @@
+package mekhq.campaign.stratcon;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StratconRulesManagerTest {
```

Comment:
I don't see a point in including an empty unit test; let's take this file out.

Reply:
Yeah, apologies, that was me attempting to (and failing to) learn how to do unit tests. I'll rip it out tomorrow, when I resolve your other comment.